sims/verilator/Makefile

```diff
@@ -127,13 +193,13 @@ model_mk_debug = $(model_dir_debug)/V$(VLOG_MODEL).mk
 $(model_mk): $(sim_vsrcs) $(sim_common_files) $(EXTRA_SIM_REQS)
 	rm -rf $(model_dir)
 	mkdir -p $(model_dir)
-	$(VERILATOR) $(VERILATOR_OPTS) -o $(sim) -Mdir $(model_dir) -CFLAGS "-include $(model_header)"
+	$(VERILATOR) $(VERILATOR_OPTS) $(EXTRA_SIM_SOURCES) -o $(sim) -Mdir $(model_dir) -CFLAGS "-include $(model_header)"
```

Comment:
There are fancy ways(used in the vlsi directory) to DRY the flags that depend on whether you are building a debug simulator or not, but perhaps we should do that in a separate PR.

Reply:
+1 to new PR's. I think this PR is getting larger by the min.

Reply:
It's clearly a fraught refactoring, given the previous incarnation had even more feedback. Thanks for working on it.